conduit-frontend/frontend-decompose-logic/src/commonTest/kotlin/mikufan/cx/conduit/frontend/logic/component/main/MainNavStoreTest.kt

```diff
@@ -0,0 +1,322 @@
+package mikufan.cx.conduit.frontend.logic.component.main
+
+import com.arkivanov.mvikotlin.core.rx.observer
+import com.arkivanov.mvikotlin.core.store.Store
+import com.arkivanov.mvikotlin.logging.store.LoggingStoreFactory
+import com.arkivanov.mvikotlin.main.store.DefaultStoreFactory
+import dev.mokkery.answering.returns
+import dev.mokkery.every
+import dev.mokkery.mock
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.flow.receiveAsFlow
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.runTest
+import mikufan.cx.conduit.frontend.logic.repo.kstore.UserConfigKStore
+import mikufan.cx.conduit.frontend.logic.repo.kstore.UserConfigState
+import mikufan.cx.conduit.frontend.logic.repo.kstore.UserInfo
+import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+import kotlin.test.assertTrue
+import kotlin.test.assertFalse
+
+class MainNavStoreTest {
+  private val testDispatcher = StandardTestDispatcher()
+  private val userConfigStateChannel = Channel<UserConfigState>(Channel.UNLIMITED)
```

Comment:
let's also make it a `private lateinit var`, and we create the new channel everytime in `@BeforeTest`